docs/docs/api.md

```diff
@@ -0,0 +1,357 @@
+---
+title: REST API
+---
+
+# REST API
+
+You can communicate with Chronos using a RESTful [JSON][] API over HTTP.
+Chronos nodes usually listen on `port 8080` for API requests.
+All examples in this section assume that you've found a running leader at `chronos-node.airbnb.com:8080`.
+
```

Comment:
Not you, but can we brand this as chronos-node.mesosphere.com:4400. Let's change ports from 8080 to 4400 as the standard since 8080 is typically used by Marathon.